R/snapshot.R

```diff
@@ -22,19 +22,19 @@
 #'
 #' @section Workflow:
 #' The first time that you run a snapshot expectation it will run `x`,
-#' capture the results, and record in `tests/testthat/snap/{test}.json`.
+#' capture the results, and record them in `tests/testthat/_snap/{test}.md`.
```

Comment:
Shouldn't this actually be `_snaps`? 
E.g., [here](https://github.com/r-lib/testthat/tree/main/tests/testthat/_snaps).

Reply:
😆 good catch! Thank you @IndrajeetPatil